actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -897,16 +897,15 @@ def sanitize_to_id(name)
         end
 
         def set_default_disable_with(value, tag_options)
-          return unless ActionView::Base.automatically_disable_submit_tag
           data = tag_options["data"]
```

Comment:
```suggestion
          data = tag_options.fetch("data", {})
```
This will save us from checking `data && data["disable_with"] == false` and also `data.delete("disable_with") if data` and `disable_with_text ||= data["disable_with"] if data`.

Reply:
@KapilSachdev this broke our app. We have places where we're passing `data: nil`:

```haml
- data = { confirm: "sure?" } if doc.confirm?
= f.submit 'Update choice', data: data
```